hack/commit-msg-verify.sh

```diff
@@ -1,21 +1,42 @@
 #!/usr/bin/env bash
 
+# This regex matches the pull number with an author from the job spec. Prow does not have a PULL_AUTHOR job var
+export PULL_AUTHOR=$(echo $JOB_SPEC | grep -Po "$PULL_NUMBER,\"author\":\"\K[^\"]*")
```

Comment:
@hunterkepley I don't think you need the `export` the `PULL_AUTHOR` variable is local to this script 👍